mllib/src/main/scala/org/apache/spark/ml/evaluation/BinaryClassificationEvaluator.scala

```diff
@@ -28,7 +28,8 @@ import org.apache.spark.sql.functions._
 import org.apache.spark.sql.types.DoubleType
 
 /**
- * Evaluator for binary classification, which expects two input columns: rawPrediction and label.
+ * Evaluator for binary classification, which expects input columns rawPrediction, label and
```

Comment:
nit: Evaluator for binary classification, which expects input columns**:** rawPrediction, label and

Reply:
Text is OK as is I think, but could quote the column names

Reply:
I guess I will leave this as is if that's OK by you two. The other two classes ```MulticlassClassificationEvaluator``` and ```RegressionEvaluator``` don't quote the column names either. If I change, I will have to change all of them.